go-controller/pkg/ovn/master.go

```diff
@@ -333,6 +333,13 @@ func (oc *Controller) StartClusterMaster() error {
 
 // SetupMaster creates the central router and load-balancers for the network
 func (oc *Controller) SetupMaster(existingNodeNames []string) error {
+	// Create default Control Plane Protection (COPP) entry for routers
+	var err error
+	oc.defaultCOPPUUID, err = EnsureDefaultCOPP(oc.nbClient)
```

Comment:
Should we have different COPP for the distributed router and the gateway routers?

It could be that we need more aggressive value for the cluster router as opposed to gateway routers?

Reply:
@girishmg Ideally we can have more fine-grained settings, but I am not sure if there is any obvious benefit yet, since the meters are applied for each node separately. So this patch simply reuses the default hard-coded CoPP. In the future there may be a need to support configurable values, and we can enhance when that requirement comes.

Reply:
@hzhou8 reached out to me on slack and I am happy with his response..

> I thought about this and I think it could be more fine-grained in the future. For now I think the default CoPP is sufficient for both. The enforcement happens on each node anyway. The 25 pps wouldn’t do any harm to the CPU. So, in theory we could create a different set of meters and CoPP for the cluster router, but it doesn’t seem to provide much value. What do you think?